core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -665,29 +665,37 @@ class HttpGoogleServicesDAO(
     } )
   }
 
-  override def getGenomicsOperation(jobId: String): Future[Option[JsObject]] = {
+  override def getGenomicsOperation(opId: String): Future[Option[JsObject]] = {
     implicit val service = GoogleInstrumentedService.Genomics
-    val opId = s"operations/$jobId"
-    val genomicsApi = new Genomics.Builder(httpTransport, jsonFactory, getGenomicsServiceAccountCredential).setApplicationName(appName).build()
-    val operationRequest = genomicsApi.operations().get(opId)
-
-    retryWithRecoverWhen500orGoogleError(() => {
-      // Google library returns a Map[String,AnyRef], but we don't care about understanding the response
-      // So, use Google's functionality to get the json string, then parse it back into a generic json object
-      Option(executeGoogleRequest(operationRequest).toPrettyString.parseJson.asJsObject)
-    }) {
-      // Recover from Google 404 errors because it's an expected return status.
-      // Here we use `None` to represent a 404 from Google.
-      case t: HttpResponseException if t.getStatusCode == StatusCodes.NotFound.intValue => None
+    val genomicsServiceAccountCredential = getGenomicsServiceAccountCredential
+    genomicsServiceAccountCredential.refreshToken()
+
+    if (opId.startsWith("operations")) {
+      // PAPIv1 ids start with "operations". We have to use a direct http call instead of a client library because
+      // the client lib does not support PAPIv1 and PAPIv2 concurrently.
+      new GenomicsV1DAO().getOperation(opId, OAuth2BearerToken(genomicsServiceAccountCredential.getAccessToken))
+    } else {
```

Comment:
meh... I don't really want to. The only reason there is a `GenomicsV1DAO` is because it has to extend `DsdeHttpDAO`. I understand the desire for symmetry but I anticipate the v1 dao going away soon anyway as google will shut down v1.